kod/object/active/holder/room/duke3.kod

```diff
@@ -100,6 +101,15 @@ messages:
 
       Send(self,@NewHold,#what=Create(&Throne),
            #new_row=10,#new_col=20,#fine_row=32,#fine_col=32,#new_angle=ANGLE_SOUTH);
+
+      oNews = Send(SYS,@FindNewsByNum,#num=NID_TERRITORY_NEWS);
+      if oNews = $
+      {
+         oNews = Create(&TerritoryNews,#nid=NID_TERRITORY_NEWS);
+      }
+      Send(self,@NewHold,#what=oNews,#new_row=15,#new_col=16,
```

Comment:
This should probably be inside the if?  Otherwise if the newsgroup already exists, NewHold will be called, potentially causing problems.